EcoSoapBank/Custom Views/KeyboardHandlingViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  KeyboardHandlingViewController.swift
+//  Catfish
+//
+//  Created by Shawn Gee on 4/4/20.
+//  Copyright © 2020 James Pacheco. All rights reserved.
+//
+
+import UIKit
+import Combine
+
+
+/// A view controller that expands to fit content vertically
+/// and responds to keyboard showing in order to not obscure text inputs.
+///
+/// Add all of your subviews to contentView. If your content should extend beyond the size of the scrollview,
+/// be sure to fully define the contentView's height through use of constraints.
+/// This view controller also adds an inputAccessoryView to each textField or textView in the content view.
+/// This facilitates easy navigation between text inputs.
```

Comment:
This should be updated at some point, at the very least to take out the reference to inputAccessoryView